app/server/api.py

```diff
@@ -202,42 +238,70 @@ class AnnotationList(generics.ListCreateAPIView):
     permission_classes = (IsAuthenticated, IsProjectUser)
 
     def get_serializer_class(self):
-        project = get_object_or_404(Project, pk=self.kwargs['project_id'])
+        project = get_object_or_404(Project, pk=self.kwargs["project_id"])
         self.serializer_class = project.get_annotation_serializer()
 
         return self.serializer_class
 
     def get_queryset(self):
-        project = get_object_or_404(Project, pk=self.kwargs['project_id'])
-        document = project.documents.get(id=self.kwargs['doc_id'])
+        project = get_object_or_404(Project, pk=self.kwargs["project_id"])
+        document = project.documents.get(id=self.kwargs["doc_id"])
         self.queryset = document.get_annotations()
         self.queryset = self.queryset.filter(user=self.request.user)
 
         return self.queryset
 
     def perform_create(self, serializer):
-        doc = get_object_or_404(Document, pk=self.kwargs['doc_id'])
+        doc = get_object_or_404(Document, pk=self.kwargs["doc_id"])
         serializer.save(document=doc, user=self.request.user)
 
 
 class AnnotationDetail(generics.RetrieveUpdateDestroyAPIView):
     permission_classes = (IsAuthenticated, IsProjectUser, IsOwnAnnotation)
 
     def get_serializer_class(self):
-        project = get_object_or_404(Project, pk=self.kwargs['project_id'])
+        project = get_object_or_404(Project, pk=self.kwargs["project_id"])
         self.serializer_class = project.get_annotation_serializer()
 
         return self.serializer_class
 
     def get_queryset(self):
-        document = get_object_or_404(Document, pk=self.kwargs['doc_id'])
+        document = get_object_or_404(Document, pk=self.kwargs["doc_id"])
         self.queryset = document.get_annotations()
 
         return self.queryset
 
     def get_object(self):
         queryset = self.filter_queryset(self.get_queryset())
-        obj = get_object_or_404(queryset, pk=self.kwargs['annotation_id'])
+        obj = get_object_or_404(queryset, pk=self.kwargs["annotation_id"])
         self.check_object_permissions(self.request, obj)
 
         return obj
+
+
+class SuggestedTerms(generics.ListAPIView):
```

Comment:
@omriallouche Could you please review this one? It working, but i'm not getting suggested words in response, only blank dictionaries. maybe i'm badly decoding it?